internal/app/secretless/listeners/ssh/listener.go

```diff
@@ -17,20 +16,15 @@ import (
 	"github.com/conjurinc/secretless-broker/internal/pkg/util"
 	"github.com/conjurinc/secretless-broker/pkg/secretless/config"
 	plugin_v1 "github.com/conjurinc/secretless-broker/pkg/secretless/plugin/v1"
-	validation "github.com/go-ozzo/ozzo-validation"
+	"github.com/go-ozzo/ozzo-validation"
```

Comment:
Third-party imports should be between local and built-in ones (even though it wasn't you that added it originally)